scripts/linters/pre_commit_linter.py

```diff
@@ -236,6 +236,7 @@ def _get_linters_for_file_extension(file_extension_to_lint, namespace, files):
         general_files_to_lint, file_cache)
     custom_linters.append(custom_linter)
 
+    config_path_for_css = os.path.join(parent_dir, 'oppia', '.stylelintrc')
```

Comment:
Can we define this inside the `css_linter` file and consider this the only config file now, I think we can avoid having `config_path` param in the `css_linter.get_linters`?

Reply:
Right.